README.md

```diff
@@ -50,7 +50,7 @@ async fn main() {
     // run our app with hyper
     // `axum::Server` is a re-export of `hyper::Server`
     let addr = SocketAddr::from(([127, 0, 0, 1], 3000));
-    tracing::debug!("listening on {}", addr);
+    tracing::info!("listening on {}", addr);
```

Comment:
I don't see much reason to change this and its inconsistent with the other example. I think we should just keep it like it was.

Reply:
Sure, I thought it was just a little more new user friendly since it doesn't print anything by default with the suggested cargo run

Either way is fine with me